src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -57,3 +57,51 @@
 		</div>
 	</div>
 </div>
+
+<modal #iterationAssociationModal title="Associate with Iteration">
+    <modal-content>
+        <div class="col-md-12 col-sm-12 col-xs-12">
+            <div *ngIf="isAssociated">
```

Comment:
Yes, it'll be used while implementing the interaction (working on expanding this patch) and with API to fetch iterations.